odrive_botwheel_explorer/description/urdf/diffbot.ros2_control.xacro

```diff
@@ -18,12 +18,25 @@
       </xacro:if>
       <joint name="${prefix}left_wheel_joint">
         <param name="node_id">0</param>
+        <command_interface name="velocity"/>
+        <command_interface name="position"/>
+        <command_interface name="torque"/>
```

Comment:
`torque` needs to be renamed to `effort` (4x). Otherwise, when I try it with the real hardware interface, it crashes with:
```
[ros2_control_node-1] terminate called after throwing an instance of 'std::runtime_error'
[ros2_control_node-1]   what():  Wrong state or command interface configuration.
[ros2_control_node-1] missing state interfaces:
[ros2_control_node-1] ' left_wheel_joint/torque '       ' right_wheel_joint/torque '
[ros2_control_node-1] missing command interfaces:
[ros2_control_node-1] ' left_wheel_joint/torque '       ' right_wheel_joint/torque '
```